soh/soh/Enhancements/randomizer/randomizer_item_tracker.cpp

```diff
@@ -290,15 +291,15 @@ bool IsValidSaveFile() {
 }
 
 bool HasSong(ItemTrackerItem item) {
-    return (1 << item.id) & gSaveContext.inventory.questItems;
+    return GameInteractor::IsSaveLoaded() ? false : ((1 << item.id) & gSaveContext.inventory.questItems);
```

Comment:
this always returns false when `IsSaveLoaded` is true?

Reply:
Oh, yeah, I had those backwards. Also, I don't think the next two are supposed to be checking for `!= 0`...

Reply:
I mean, it works, but that seems incidental.

Reply:
I had them ALL backwards XD. Thanks for catching that.